octobot_evaluators/channels/evaluators.py

```diff
@@ -0,0 +1,135 @@
+# cython: language_level=3
+#  Drakkar-Software OctoBot-Evaluators
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+from octobot_channels.channels.channel import Channel
+from octobot_channels.constants import CHANNEL_WILDCARD
+from octobot_channels.consumer import Consumer
+from octobot_channels.producer import Producer
+from octobot_commons.logging.logging_util import get_logger
+
+
+class EvaluatorsChannelConsumer(Consumer):
+    pass
+
+
+class EvaluatorsChannelProducer(Producer):
+    # noinspection PyMethodOverriding
+    async def send(self,
+                   matrix_id,
+                   data,
```

Comment:
Should we keep the same arguments order in send() and the consumers callback ? data as the last argument ?

Reply:
Yes it should be the last argument but I put it here because it's the last mandatory argument after matrix_id. To respect the python way of declaring argument I put the mandatory arguments first. I can also make data optionnal and put it as the last argument.

Reply:
Right, it should be mandatory !